enterprise/api/v1/urls.py

```diff
@@ -1,22 +1,17 @@
 """
 URL definitions for enterprise api version 1 endpoint.
 """
+
 from __future__ import absolute_import, unicode_literals
 
 from rest_framework.routers import DefaultRouter
 
 from enterprise.api.v1 import views
 
-# Pylint considers module level variable as "Constant" and expects them to be upper-cased
-# that is why we have disabled 'invalid-name' check for variable definition below.
```

Comment:
We disable invalid-name in other places, too, and I assume most people know the reason (roughly). Don't need 2 full lines explaining it.